solidity/random-beacon/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.0-dev",
   "description": "Keep Random Beacon",
   "scripts": {
+    "clean": "rm -rf build && rm -rf cache && rm -rf typechain",
```

Comment:
`hardhat clean` seems to do exactly that, so maybe:
`"clean": "hardhat clean"` ?

Reply:
Good point! See https://github.com/keep-network/keep-core/pull/2671/commits/aacaaa1e1e096f4c24d77dba19b35bb917aa4d19